pkg/volume/plugins.go

```diff
@@ -89,6 +89,14 @@ type RecyclableVolumePlugin interface {
 	NewRecycler(spec *Spec) (Recycler, error)
 }
 
+// CreatableVolumePlugin is an extended interface of VolumePlugin and is used to create persistent volumes for the cluster.
+type CreatableVolumePlugin interface {
+	VolumePlugin
+	// NewCreater creates a new volume.Creater which knows how to create this resource
+	// in accordance with the underlying storage provider
+	NewCreater(spec *Spec) (Creater, error)
```

Comment:
There won't be a pod when a PV is being created.

Security is important, though, and this constructor func was changed to work exactly like Builder, which takes VolumeOptions.  We've got access to the same security info on the options that Builder has.